src/layouts/index.jsx

```diff
@@ -1,51 +1,154 @@
-import { Menu, Search, Nav, Shell, Radio } from '@alicloudfe/components';
-import '@alicloudfe/components/dist/hybridcloud.css';
+import React, { useState, useEffect } from 'react';
+import {
+  Layout,
+  Menu,
+  Breadcrumb,
+  Divider,
+  Avatar,
+  Dropdown,
+  Button,
+} from 'antd';
+import { SettingOutlined } from '@ant-design/icons';
```

Comment:
Use `import { Icon } from 'antd';`

Reply:
antd 4.x update icon to @ant-design/icons